index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Abel Tefera | Software Engineer</title>
+  </head>
+  <body>
+    <nav class="header">
+      <span id="logo">Abel Tefera</span>
+      <div class="hamburger">
+        <img src="assets/hamburger.svg" />
+    </div>
+      <div class="links-container">
+        <a class="links" href="#">Portfolio</a>
+        <a class="links" href="#">About</a>
+        <a class="links" href="#">Contact</a>
+      </div>
+    </nav>
+    <div class="container">
+        <h1 id="headline">I'm Abel, <br />Glad to see you!</h1>
+        <p id="description">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what you
+          see and have a project you need coded, don’t hestiate to contact me.
+        <p>
+        <h5 id="subtitle">LET'S CONNECT</h5>
+        <div class="social-links">
+            <img src="assets/twitter.svg" />
+            <img src="assets/medium.svg" />
+            <img src="assets/github.svg" />
+            <img src="assets/angellist.svg" />
+        </div>
```

Comment:
Kindly update this element to use a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`